### PR TITLE
Ide 2088 2

### PR DESCRIPTION
**Pull request by AndyWu2015**

### CODE REVIEW

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java [hunk 1]
```diff
@@ -122,10 +120,6 @@ public void beginTask( String taskName, int totalWork )
                             }
                         };
 
-                        final Dictionary<String, Integer> properties = new Hashtable<>();
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java]: Why did you remove this code?  Can you put it back?

**Comment by gamerson** [comment]
Closing see one code comment and also can you change your 2nd commit message to some different message than the first one?  thanks.

**Comment by gamerson** [comment]
Merged, thanks! See 411c5d3...6023b1b
411c5d3...6023b1b <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by AndyWu2015** [comment]
Sorry I will remember to change commit message next time .
